_episodes_rmd/01-r-plotting.Rmd

```diff
@@ -425,15 +425,22 @@ ggplot(data = gapminder_1997) +
 
 OK. That looks better. Notice that when we added the label value we did so by placing the values inside quotes. This is because we are not using a value from inside our data object - we are providing the name directly. When you need to include actual text values in R, they will be placed inside quotes to tell them apart from other object or variable names.
 
-Let's now map our `lifeExp` values to the y axis and give them a nice label.
-
-```{r FirstPlotAddY}
-ggplot(data = gapminder_1997) +
-  aes(x = gdpPercap) +
-  labs(x = "GDP Per Capita") +
-  aes(y = lifeExp) +
-  labs(y = "Life Expectancy")
-```
+> ## Exercise: Reading in an excel file
```

Comment:
This doesn't seem like the right exercise title. :)

Reply:
Oops yes! Just fixed it